packages/light-apps/src/types.ts

```diff
@@ -13,12 +13,11 @@ export type Route = {
   icon: SemanticICONS,
   isApiGated: boolean,
   isHidden: boolean,
```

Comment:
I was thinking to remove the 3 field above, because I believe they are not needed. Any thoughts @yjkimjunior?

Reply:
I was thinking icon could still be used though not strictly necessary. `isApiGated` and `isHidden` almost definitely won't be used so I'm fine with removing them. 👍